superset/db_engine_specs.py

```diff
@@ -529,6 +529,17 @@ def truncate_label(cls, label):
             label = label[:cls.max_column_name_length]
         return label
 
+    @staticmethod
+    def get_timestamp_column(expression, column_name):
+        """Return the expression if defined, otherwise return column_name. Some
+        engines require forcing quotes around column name, in which case this method
+        can be overridden."""
+        return expression or column_name
```

Comment:
Is this a remnant from some earlier state? I don't see this method being called anywhere.

Reply:
oh right, bad merge conflict resolve... removing